core/src/model/ModelFactory.java

```diff
@@ -1,15 +1,15 @@
 package model;
 
-import model.enemies.*;
+import model.monsters.*;
```

Comment:
Every hard coded monster should be deleted in order to merge with the rest of the code